### DISCUSSION TITLE
Unsubscribing from an issue accidentally (and missing the "undo" button) leads to irreversible loss of data flow

### DISCUSSION THREAD

**Discussion by peppy** [community/community#3739] [Mobile] [answered]
I accidentally swiped to unsubscribe on an issue in my notification box. I don't know which issue it was and I missed the undo button (was during multitasking on iPadOS). As far as I can tell, there's no way for me to find out which issue I unsubscribed from, meaning it will be silenced to my view forever.

Pretty frustrating UX as it stands. Being able to hit Cmd-z multiple times to undo (or something similar) would be appreciated.

**Comment by brianlovin** [comment, marked as answer]
Thanks for the feedback here, and sorry for the confusing UX. Right now when you unsubscribe from a notification, it will go to the Done view of your notifications. Tapping `Filter` in the navigation bar, then `Done` will show you a list of all your notifications that were marked as done or unsubscribed.

**Reply by peppy** [reply]
Oh, thanks for the information. I actually had no idea that unsubscribed notifications also end up in done. Will remember that for next time!

**Reply by peppy** [reply]
This doesn't look to be correct information:

![20210415 150055 (Safari)](https://user-images.githubusercontent.com/191335/114821382-71692380-9dfb-11eb-93c3-c8b11b04e768.gif)

Am I missing something?

**Reply by brianlovin** [reply]
It's because things in Done are sorted by recent activity. The individual notification you marked as Done was updated `Yesterday` and then you had a lot more recent stuff at the top of Done that had been active in the most recent hour or two. It is confusing that it works this way, sorry about that.

**Reply by peppy** [reply]
Hmm, that may still make it hard to recover from the scenario I mentioned then. I keep things in my inbox for months, sometimes, so it would likely be impossible to find marked items in the "done" area unless sorting by the time of state change, which doesn't look possible.

**Reply by brianlovin** [reply]
I'll flag this to the team. Perhaps Done should just be sorted by the timestamp where the notification was marked as Done?

**Reply by peppy** [reply]
I think that would be a huge improvement, and also what most users visiting the page would expect in the context of a notifications system.

**Reply by PSSF23** [reply]
When it's "unsubscribe & done" the swipe action should be labeled clearly.

If it says "unsubscribe" then it should **only** unsubscribe as stated! Not anything else.

**Reply by PSSF23** [reply]
And the current "done" folder is useless. No filter of any kind. I keep all notifications in inbox because they can be filtered by repository there.

**Reply by jayaddison** [reply]
@brianlovin Thank you - I'm very glad to learn that unsubscribed notifications are moved into the 'done' list, after accidentally unsubscribing from a thread recently.

If larger user interface changes for this are pending / more difficult, would it be OK to suggest an explanatory message to explain that unsubscriptions are placed into the 'done' notifications filter?

For example, when the following occurs:

![image](https://user-images.githubusercontent.com/55152140/228491795-07124978-60ee-4410-befa-9215cd184829.png)
(alt text: an image with a toast notice displaying the text: "You have been unsubscribed from 2 threads.")

... then perhaps the message could be suffixed with "You can find the unsubscribed threads in the ['Done' category](https://github.com/notifications?query=is%3Adone) of your notifications".
